supplementary_style_guide/style_guidelines/graphical-interfaces.adoc

```diff
@@ -29,7 +29,9 @@ In the *Name* field, enter `test-postgresql`.
 [[user-interface-elements]]
 == User interface elements
 
-Bold all graphical user interface elements, including, but not limited to, menu names, menu items, button names, dialog box names, and window names. Bold the item, even if it is not clickable.
+Bold all graphical user interface (GUI) element names, including, but not limited to, menu, menu item, button, dialog box, and window names. Bold the element name if it appears in the GUI, even if it is not clickable.
```

Comment:
```suggestion
Use bold text for all graphical user interface (GUI) element names, including, but not limited to, menu, menu item, button, dialog box, and window names. Use bold text for the element name if the name appears in the GUI, even if the element is not clickable.
```

A few minor comments:

1. Bold is an adjective, not a verb. :) 
2. Some of the pronouns were ambiguous -- please check my suggested rewrites, because I might have misinterpreted the original pronoun targets!

Reply:
I thought about "bold" as a verb - went as far as checking merriam webster :smile: But I couldn't think of a better alternative and figured I'd leave it alone since it was already there. Thanks for making a great suggestion!